submitted_models/coro_jeanine_sensor_config_1/launch/vehicle_topics.launch

```diff
@@ -0,0 +1,276 @@
+<?xml version="1.0"?>
+<launch>
```

Comment:
the `IGN_TRANSPORT_TOPIC_STATISTICS` env var is msising:

```
<env name="IGN_TRANSPORT_TOPIC_STATISTICS" value="1"/>
```

from step 8: https://github.com/osrf/subt/wiki/Model%20Submission%20Process#adding-a-custom-configuration-to-existing-vehicles

Reply:
I'll make this change in a follow up pr with marsupials pairs.